src/jamf_pro_sdk/models/pro/scripts.py

```diff
@@ -0,0 +1,28 @@
+from typing import Optional
+
+from pydantic import Extra
+
+from .. import BaseModel
+
+
+class Script(BaseModel, extra=Extra.allow):
+    """Represents a script record. The same data is
+    returned for both GET scripts and GET script by ID."""
+
+    id: Optional[int]
+    name: Optional[str]
+    info: Optional[str]
+    notes: Optional[str]
+    priority: Optional[str]
```

Comment:
This should be an enum.
```suggestion
    priority: Optional[ScriptPriorities]
```
Enum model to add to this file:
```python
class ScriptPriority(str, Enum):
    BEFORE: str = "BEFORE"
    AFTER: str = "AFTER"
    AT_REBOOT: str = "AT_REBOOT"
```